server/hydrogen-render/render-hydrogen-vm-render-script-to-page-html.js

```diff
@@ -48,6 +48,8 @@ async function renderHydrogenVmRenderScriptToPageHtml(
           <meta name="viewport" content="width=device-width, initial-scale=1">
           ${maybeNoIndexHtml}
           ${sanitizeHtml(`<title>${pageOptions.title}</title>`)}
+          <link rel="icon" type="image/png" href="/assets/images/favicon.png">
```

Comment:
48x48 is what https://realfavicongenerator.net/svg-favicon/ spat out but I suspect it might be better to use a 64x64 image with a white background